src/dynamic_sharing_service/policy/source.constraint.builder.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const SourceConstraint = require('./source.constraint');
+const WantedAnchorBuilder = require('./wanted.anchor.builder');
+
+const ANCHOR_PASSPORT = 'PASSPORT';
```

Comment:
I think I commented on the Python PR the same thing, but I think these should live in the same place as where other constants are kept (i.e. where we keep the attribute constants).  That way if we need access somewhere else in the SDK, we don't have to maintain it in two places

Reply:
I believe I discussed this with @echarrod - we have the helper methods for these document types, so there's not really a need to expose the constants at the moment.

However, it might help someone using the `withAnchorByValue()` method - @echarrod what do you think?

We'd probably want to make them more descriptive if we do this, somethinng like`ANCHOR_VALUE_PASSPORT`

Reply:
Would there be a reason that they'd use `withAnchorByValue("Passport")`, rather than the helper method?

If not, I think we can keep them here, and then we can always move them to the common constants file if we need them elsewhere (assuming they're not exported here).

Also renaming to `ANCHOR_VALUE_PASSPORT` seems sensible, even if we don't move it

Reply:
Renamed in c32df35ea3b3922699d23a984bc526b4ae33eddc